docs/topics/proc_add-tomcat-logging.adoc

```diff
@@ -0,0 +1,82 @@
+= Add Tomcat Logging
+
+ This example adds configuration necessary to send the Embedded Tomcat containers’ access logging to stdout.
+
+
+.Prerequisites
+* An application you want to enable Tomcat logging for. For example, xref:mission-http-api-spring-boot[a booster].
+*
+
+.Procedure
+
+.  Set `server.use-forward-headers` to  `true` in your application.yml file:
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+server:
+  use-forward-headers: true
+----
+
+. Specify logging levels (e.g. DEBUG) for the  ` org.apache.tomcat` package and specify the name of the logging file.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+logging:
+  level.org.apache.tomcat: "DEBUG"
+  file: "application.log"
+----
+
+. Configure your embedded Tomcat container and add `logback-access` by defining a provider method.
```

Comment:
This should read something like "Add a `LogbackValve` to your embedded Tomcat container to use the Logback logging framework."

Reply:
@Ladicek will update  comment per your suggestion.